.dotnet/tests/Samples/Chat/Sample01_SimpleChat.cs

```diff
@@ -12,10 +12,10 @@ public void Sample01_SimpleChat()
         {
             ChatClient client = new("gpt-3.5-turbo", Environment.GetEnvironmentVariable("OPENAI_API_KEY"));
 
-            ChatCompletion chatCompletion = client.CompleteChat("How does AI work? Explain it in simple terms.");
+            ChatCompletion chatCompletion = client.CompleteChat([new UserChatMessage("How does AI work? Explain it in simple terms.")]);
```

Comment:
Not a thing for this PR -- this is where I'm curious if we could safely add an implicit operator to treat strings as user-mode messages with simple string content, enabling this to just be `client.CompleteChat(["How does AI work?"]);`